data/184/752/401/9/1847524019.geojson

```diff
@@ -0,0 +1,55 @@
+{
+  "id": 1847524019,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.017784,
+    "geom:bbox":"8.419907,53.454357,10.166872,53.964403",
+    "geom:latitude":53.54225,
+    "geom:longitude":9.948146,
+    "iso:country":"DE",
+    "lbl:latitude":53.51805,
+    "lbl:longitude":10.013092,
+    "mps:latitude":53.51805,
+    "mps:longitude":10.013092,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:deu_x_preferred":[
```

Comment:
For DEU and ENG `Mitte` should be preferred name, and `Hamburg-Mitte` a variant (and possibly a long name for disambiguation). Also consider modifying wof:name.